internal/validations/validations.go

```diff
@@ -288,6 +290,23 @@ func validateNotStatic(_ context.Context, _ string, baton *Baton) *types.Validat
 	return types.NewValidationError(types.ErrNotDynLinked)
 }
 
+func validateExcludedCryptoModules(ctx context.Context, path string, baton *Baton) *types.ValidationError {
+	var stdout bytes.Buffer
+	cmd := exec.CommandContext(ctx, "nm", "-j", path)
+	cmd.Stdout = &stdout
+	if err := cmd.Run(); err != nil {
+		return types.NewValidationError(err)
+	}
+
+	// Make this more flexible by deriving the excluded modules from
+	// configuration.
+	excluded := []byte("golang.org/x/crypto")
```

Comment:
Ideally - this could be a two-dimensional array that we can load from configuration.

However, that will mean we need to duplicated the excluded modules for each release.

We could invert the logic to opt-into excluding x/crypto by default, and having a configuration option to opt out.